### PR TITLE
Update gradle from 2.3.2 to 2.3.3

### PR DESCRIPTION
**Pull request by ericcornelissen**
- Relevant Issues: -
- Related Pull Requests: #288 

* * *

## What
This Pull Request updates the gradle version used in this repository from 2.3.2 to 2.3.3

## Why
This Pull Request is needed because we want to keep our tools up to date, especially our main build tool 😄

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.2'
+        classpath 'com.android.tools.build:gradle:2.3.3'
```
Comment (TheBonheurs) [inline build.gradle]: Insane code hack right here, props man 😂

Comment (ericcornelissen) [inline build.gradle]: 😆

**Comment by jverbraeken** [comment]
@ericcornelissen You have to update .travis.yml as well 😉

**Comment by ericcornelissen** [comment]
Updated this PR so it only changes the gradle version from v2.3.2 to v2.3.3. The changes to the APK will come in a separate PR 😄

**Pull request merged**